### ISSUE TITLE
Demoapp: Detect user-defined layout before usage

### ISSUE THREAD

**Issue by EdwardCoyle** [infor-design/enterprise#1357]
This is a follow-up issue to #1327.  It's currently possible to use query params to set the desired alternate layout for a test/example page, but the existence of the layout is not validated before the demoapp attempts to load it.

**Describe the bug**
For example, running http://localhost:4000/components/dropdown/example-index?layout=notalayout currently errors out with a 404.  The error happens on the attempt to load the layout from the filesystem.

**Expected behavior**
We should find a way to detect if the layout exists or not, defaulting to the standard one that it would otherwise load if no match exists.  In this case, the query parameter will simply be ignored if it doesn't match a valid layout. We don't have anything that scans the filesystem for a matching layout at the moment so we'll need to write a small detector for the file.

**Version**
- Starting in v4.13.x
- n/a in prior versions

**Comment by brianjuan** [comment]
Passed QA testing on localhost, will do further testing once 4.15.0 build is ready.
